nfd/instance/overlays/only-nvidia/patch-node-feature-discovery.yaml

```diff
@@ -0,0 +1,20 @@
+- op: add
```

Comment:
This might work better as an object that uses the merge patch strategy instead of adding the configuration.

Similar to this:
https://github.com/redhat-cop/gitops-catalog/blob/main/openshift-gitops-operator/instance/components/edge-termination/patch-edge-termination.yaml

Reply:
IDK what I was thinking... But my way was way more complicated :smile: